tool/tsh/common/mcp_db.go

```diff
@@ -226,9 +233,175 @@ func (c *mcpDBStartCommand) prepareDatabases(
+		tc, err := makeClient(m.cf)
+		if err != nil {
+			return trace.Wrap(err)
+		}
+
+		m.databasesGetter = tc
+		m.siteName = tc.SiteName
+	}
+
+	databases, err := m.databasesGetter.ListDatabases(m.ctx, &proto.ListResourcesRequest{
+		Namespace:           apidefaults.Namespace,
+		ResourceType:        types.KindDatabaseServer,
+		PredicateExpression: makeDiscoveredNameOrNamePredicate(m.cf.DatabaseService),
+		// TODO(gabrielcorado): support requesting access.
+		UseSearchAsRoles: false,
+	})
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	db, err := chooseOneDatabase(m.cf, databases)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	// TODO(gabrielcorado): support having the flags empty and assume the values
+	// based on the role and database.
+	if m.cf.DatabaseUser == "" || m.cf.DatabaseName == "" {
+		return trace.BadParameter("You must specify --db-user and --db-name flags used to connect to the database")
+	}
+
+	dbURI := mcp.NewDatabaseResourceURIWithConnectParams(m.siteName, db.GetName(), m.cf.DatabaseUser, m.cf.DatabaseName)
+	switch {
+	case m.clientConfig.isSet():
+		return trace.Wrap(m.updateClientConfig(dbURI))
+	default:
+		return trace.Wrap(m.printJSONWithHint(dbURI))
+	}
+}
+
+func (m *mcpDBConfigCommand) printJSONWithHint(dbURI mcp.ResourceURI) error {
+	config := claude.NewConfig()
+	if err := m.addDatabaseToConfig(config, dbURI); err != nil {
+		return trace.Wrap(err)
+	}
+
+	w := m.cf.Stdout()
+	if _, err := fmt.Fprintln(w, "Here is a sample JSON configuration for launching Teleport MCP servers:"); err != nil {
+		return trace.Wrap(err)
+	}
+	if err := config.Write(w, claude.FormatJSONOption(m.clientConfig.jsonFormat)); err != nil {
+		return trace.Wrap(err)
+	}
+	if _, err := fmt.Fprintln(w, ""); err != nil {
+		return trace.Wrap(err)
+	}
+	return trace.Wrap(m.clientConfig.printHint(w))
+}
+
+func (m *mcpDBConfigCommand) updateClientConfig(dbURI mcp.ResourceURI) error {
+	config, err := m.clientConfig.loadConfig()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	if err := m.addDatabaseToConfig(config, dbURI); err != nil {
+		return trace.Wrap(err)
+	}
+
+	if err := config.Save(claude.FormatJSONOption(m.clientConfig.jsonFormat)); err != nil {
+		return trace.Wrap(err)
+	}
+
+	_, err = fmt.Fprintf(m.cf.Stdout(), `Updated client configuration at:
+%s
+
+Teleport database access MCP server is named %q in this configuration.
+
+You may need to restart your client to reload these new configurations.
+`, config.Path(), mcpDBConfigName)
+	return trace.Wrap(err)
+}
+
+// addDatabaseToConfig adds the provided database, merging with existent
+// databases configured.
+func (m *mcpDBConfigCommand) addDatabaseToConfig(config claudeConfig, dbURI mcp.ResourceURI) error {
+	var (
+		dbs     []string
+		updated bool
+		server  = makeLocalMCPServer(m.cf, nil /* args */)
+	)
+	if existentServer, ok := config.GetMCPServers()[mcpDBConfigName]; ok {
+		server.Command = existentServer.Command
+		server.Envs = existentServer.Envs
```

Comment:
i think we should just overwrite these with new values instead of using old values. if we want to be on the safe side, lets check if they have changed and prompt user for overwrite.

Reply:
A new flag is also fine. Instead of `--fix`, how about `--overwrite`?

If not specified, but the content is different, print something like:
```
Environment variables have changed, but existing values will be preserved.
To overwrite them, rerun this command with the --overwrite flag.
```

Reply:
Done, I've added the flag and the message.